seedpod_ground_risk/layers/osm_tag_layer.py

```diff
@@ -207,6 +203,28 @@ def query_osm_polygons(osm_tag, bound_poly: sg.Polygon) -> gpd.GeoDataFrame:
     return poly_df
 
 
+def query_request(overpass_url, osm_tag, bounds):
+    headers = {'User-Agent': 'seedpod-ground-risk v0.13.0 (Python 3.8/requests v2.25.1;)',
```

Comment:
Set the versions programmatically eg `requests.__version__` instead of hard coding then having to up date every release!